ironfish-cli/src/commands/wallet/transactions/index.ts

```diff
@@ -62,6 +62,14 @@ export class TransactionsCommand extends IronfishCommand {
       options: ['notes', 'transactions', 'transfers'],
       helpGroup: 'OUTPUT',
     }),
+    'filter.start': Flags.string({
+      description: 'include transactions after this date (inclusive). Example: 04/20/2023',
```

Comment:
Nit: I would change the example to an ISO date (to be more understandable to an international audience)

Reply:
Done!